tests/test_skvbc_auto_view_change.py

```diff
@@ -51,16 +51,20 @@ class SkvbcAutoViewChangeTest(unittest.TestCase):
         """
         bft_network.start_all_replicas()
 
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+
         initial_primary = 0
 
         # do nothing - just wait for an automatic view change
-        await bft_network.wait_for_view_change(
+        await bft_network.wait_for_view(
             replica_id=random.choice(
                 bft_network.all_replicas(without={initial_primary})),
             expected=lambda v: v > initial_primary,
             err_msg="Make sure automatic view change has occurred."
         )
 
+        skvbc.read_your_writes(bft_network, self)
```

Comment:
By the way, no need to pass a _bft_network_ parameter here - the _SimpleKVBCProtocol_ already has one, accessible via _self.bft_network_.